ext/js/language/zh/chinese.js

```diff
@@ -73,3 +74,11 @@ export function isCodePointChinese(codePoint) {
 export function normalizePinyin(str) {
     return str.normalize('NFC').toLowerCase().replace(/[\s・:'’-]|\/\//g, '');
 }
+
+/** @type {import('language').TextProcessor<boolean>} */
+export const normalizeRadicalCharacters = {
+    name: 'Normalize radical characters',
+    description: '⼀ → 一 (U+2F00 → U+4E00)',
+    options: basicTextProcessorOptions,
+    process: (str, setting) => (setting ? normalizeRadicals(str) : str),
+};
```

Comment:
is this imported anywhere? it seems it's all being imported from japanese in language descriptors?

Reply:
Put it all in CJK-util instead now.